posthog/models/team/team.py

```diff
@@ -41,6 +41,9 @@
 
 
 class TeamManager(models.Manager):
+    def get_queryset(self):
```

Comment:
Oh, this is pretty cool!

I bet dashboards and insights would benefit from this too! 🙌 🤔